qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -161,15 +160,16 @@ def write_local_config(self):
         self.test_dev = mon_sock + ':/' + self.test_dirname
         self.scratch_dev = mon_sock + ':/' + self.scratch_dirname
 
-        xfstests_config_contents = dedent('''\
+        xfstests_config_contents = dedent(f'''\
             export FSTYP=ceph
-            export TEST_DEV={}
-            export TEST_DIR={}
-            #export SCRATCH_DEV={}
-            #export SCRATCH_MNT={}
-            export TEST_FS_MOUNT_OPTS="-o name=admin,secret={}"
-            ''').format(self.test_dev, self.test_dirs_mount_path, self.scratch_dev,
-                        self.scratch_dirs_mount_path, self.get_admin_key())
-
-        self.mount_a.client_remote.write_file(join(self.repo_path, 'local.config'),
+            export TEST_DEV={self.test_dev}
+            export TEST_DIR={self.test_dirs_mount_path}
+            export TEST_FS_MOUNT_OPTS="-o name=admin,secret={self.get_admin_key()}"
```

Comment:
`s/TEST_FS_MOUNT_OPTS/CEPHFS_MOUNT_OPTIONS/`

From the `xfstest-dev` source code:

```
394 _test_mount_opts()
395 {
396         case $FSTYP in
397         9p)
398                 export TEST_FS_MOUNT_OPTS=$PLAN9_MOUNT_OPTIONS
399                 ;;
400         cifs)
401                 export TEST_FS_MOUNT_OPTS=$CIFS_MOUNT_OPTIONS
402                 ;;
403         ceph)
404                 export TEST_FS_MOUNT_OPTS=$CEPHFS_MOUNT_OPTIONS                                                                                    
405                 ;;
406         nfs)
407                 export TEST_FS_MOUNT_OPTS=$NFS_MOUNT_OPTIONS
408                 ;;
409         glusterfs)
410                 export TEST_FS_MOUNT_OPTS=$GLUSTERFS_MOUNT_OPTIONS
411                 ;;
412         ext2|ext3|ext4|ext4dev)
413                 # acls & xattrs aren't turned on by default on older ext$FOO
414                 export TEST_FS_MOUNT_OPTS="-o acl,user_xattr $EXT_MOUNT_OPTIONS"
415                 ;;
416         *)
417                 ;;
418         esac
419 }
```
If we left the `CEPHFS_MOUNT_OPTIONS` empty will the `TEST_FS_MOUNT_OPTS` and `MOUNT_OPTIONS` be reset to empty too ?

Locally only setting `CEPHFS_MOUNT_OPTIONS` all the test cases work well for me.

Never test this config yet.